### PR TITLE
LPS-140377 | Master

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-140377

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

File: build-test.xml [hunk 1]
```diff
@@ -13647,6 +13659,15 @@ url=${short.hostname}:8080</echo>
 		<setup-testable-tomcat />
 	</target>
 
+	<target name="setup-websphere-ssl-truststore">
+		<replace
+			file="${app.server.websphere.instance.dir}/config/cells/liferay-cell/nodes/liferay-node/servers/server1/server.xml"
+		>
+			<replacetoken><![CDATA[genericJvmArguments="]]></replacetoken>
+			<replacevalue expandproperties="true"><![CDATA[genericJvmArguments="-Djavax.net.ssl.trustStore=${app.server.websphere.dir}/java/8.0/jre/lib/security/cacerts -Djavax.net.ssl.trustStorePassword=changeit -Djavax.net.ssl.trustStoreType=jks ]]></replacevalue>
```
Comment (austinchiang) [inline build-test.xml:13667]: Hey @kenjiheigel gonna summon you for your expertise 😀 do you have any concerns if we used the default TrustStore password here or should we...change it (*badumpts*)?

**Comment by austinchiang** [comment]
ci:forward

**Comment by austinchiang** [comment]
Good work @Jeremy-Ch

**Comment by austinchiang** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
ci:forward

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

**Pull request closed without merging**